src/composed/cleartext.rs

```diff
@@ -554,8 +559,13 @@ mod tests {
     fn test_sign() {
         let key_data = std::fs::read_to_string("./tests/unit-tests/cleartext-key-01.asc").unwrap();
         let (key, _) = SignedSecretKey::from_string(&key_data).unwrap();
-        let msg = CleartextSignedMessage::sign("hello\n-world-what-\nis up\n", &key, String::new)
-            .unwrap();
+        let msg = CleartextSignedMessage::sign(
+            &mut thread_rng(),
```

Comment:
ideally we would switch to a seeded rng in all tests, so that they are reproducible

Reply:
chacha8 is crypto + seeded

Reply:
I've adjusted the new tests to use ChaCha8.

While looking around, I noticed that some older tests use `StdRng` and `XorShiftRng`. Should we leave those, or normalize them to all use ChaCha8?